xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -191,7 +191,7 @@ If this change is undesirable, you can configure your application to use the old
         <ReturnType>System.Net.ICredentials</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets the credentials used to authenticate the user to an authenticating proxy.</summary>
```

Comment:
I think this can reuse the verbage for [HttpClientHandler.DefaultProxyCredentials](https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpclienthandler.defaultproxycredentials?view=netcore-3.0). @carlossanlop, is there an easy way to share this?